skimage/util/montage.py

```diff
@@ -108,3 +118,84 @@ def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
     arr_out = arr_out.reshape(alpha_y * height, alpha_x * width)
 
     return arr_out
+
+def montage_rgb(arr_in, fill='mean', grid_shape=None, border_padding=0):
```

Comment:
Do you really need this function? I have the feeling that there is a lot of duplicated code from ``montage2d``. Can't you just build the montage for each channel and then combine them together?